src/components/Illustrations/types.ts

```diff
@@ -0,0 +1,33 @@
+export interface IllustrationProps {
+    id?: string;
+    logo: chain | logos;
+
+    /**
+     * possible sizes are "xs" | "s" | "m" | "l" | "xl"
+     */
+    size?: size;
+
+    /**
+     * set inner color
+     */
+    fillInline?: string;
+
+    /**
+     * set border color
+     */
+    fillOutline?: string;
+
+    /**
+     * Use for ethereum and arbitrum only
+     */
+    fillInlineLeft?: string;
+
+    /**
+     * Use for ethereum and arbitrum only
+     */
+    fillInlineRight?: string
+}
+
+export type size = "xs" | "s" | "m" | "l" | "xl";
```

Comment:
31, 32, 33 please check the types of `LinkTo` Oscar has a cooler way to do this

Reply:
done